data/uk/scotland/west-dunbartonshire/west-dunbartonshire.csv

```diff
@@ -0,0 +1,5 @@
+id,name,brewery_type,address_1,address_2,address_3,city,county_province,postal_code,website_url,phone,created_at,updated_at,country,longitude,latitude,tags
```

Comment:
Let's keep `address_1` as `address`. So it'd be `...,address,address_2,address_3,...` . We might end up changing it to `address_1` down the road, but I want to limit things this round to additions.

Reply:
Ugh. Actually, I named the original column/field as `street`, which is worse!  They always say you should be embarrassed by your past code. I am. 🤦 😆    

So yeah, we'll update `street` to `address_1`. First, I think we need to figure out how many projects that change will break and/or give notice of the breaking change.

Reply:
I'm happy to change the address_1 to address or even street. Let me know what works best. 👍

Reply:
Let's use `street` for consistency. Thanks Andrew!

Reply:
Should be good to go!